docroot/modules/humsci/hs_layouts/hs_layouts.libraries.yml

```diff
@@ -1,3 +1,9 @@
+admin:
```

Comment:
I went the way of putting layouts in to a module for earth and found that because grids can shift and styles can change it is better to have all of these in the theme. Let's chat about this sometime.